hoomd/md/pair/aniso.py

```diff
@@ -477,7 +477,8 @@ def __init__(self, nlist, default_r_cut=None, mode='none'):
                               rounding_radii=OnlyIf(
                                   to_type_converter((float, float, float)),
                                   preprocess=self._to_three_tuple),
-                              len_keys=1))
+                              len_keys=1,
+                              _defaults = {'rounding_radii', 0.0}))
```

Comment:
```suggestion
                              _defaults = {'rounding_radii', (0.0, 0.0, 0.0)}))
```
`rounding_radii` is a tuple with 3 floats.